core/src/index.ts

```diff
@@ -7,51 +7,21 @@ import tar from 'tar';
 import logger from './utilities/logger';
 import { getAllMatches } from './utilities/string-helper';
 import { RederlyCourse, RederlyTopic, RederlyQuestion } from './models';
+import { imageInPGFileRegex, dequotePerlQuotes } from '@rederly/rederly-utils/lib/importer';
 
+// eslint-disable-next-line @typescript-eslint/no-var-requires
```

Comment:
Comment explaining that this is to support out-of-src access to the `package.json`